analytics/views/support.py

```diff
@@ -103,8 +104,43 @@ def get_confirmations(
     return confirmation_dicts
 
 
+DOWNGRADE_METHODS = [
+    "downgrade_at_billing_cycle_end",
+    "downgrade_now_without_additional_licenses",
+    "downgrade_now_void_open_invoices",
+]
+
+STATUS = [
+    "active",
+    "deactivated",
+]
+
+BILLING_METHOD = [
+    "send_invoice",
+    "charge_automatically",
+]
+
+
 @require_server_admin
-def support(request: HttpRequest) -> HttpResponse:
+@has_request_variables
+def support(
+    request: HttpRequest,
+    realm_id: Optional[str] = REQ(default=None),
+    plan_type: Optional[int] = REQ(default=None, converter=to_non_negative_int),
+    discount: Optional[Decimal] = REQ(default=None, converter=lambda s: Decimal(s)),
+    new_subdomain: Optional[str] = REQ(default=None, str_validator=check_string),
+    status: Optional[str] = REQ(default=None, str_validator=check_string_in(STATUS)),
+    billing_method: Optional[str] = REQ(
+        default=None, str_validator=check_string_in(BILLING_METHOD)
+    ),
+    sponsorship_pending: Optional[bool] = REQ(default=None, json_validator=check_bool),
+    approve_sponsorship: Optional[str] = REQ(default=None),
+    downgrade_method: Optional[str] = REQ(
+        default=None, str_validator=check_string_in(DOWNGRADE_METHODS)
+    ),
+    scrub_realm: Optional[str] = REQ(default=None),
```

Comment:
I guess same goes for scrub_realm as well. Take extra testing the change since this results in scrubbing the entire realm.